examples/air/main.py

```diff
@@ -222,7 +222,7 @@ def per_param_optim_args(module_name, param_name):
 
         if args.viz and i % args.viz_every == 0:
             trace = poutine.trace(air.guide).get_trace(examples_to_viz, None)
-            z, recons = poutine.replay(air.prior, trace)(examples_to_viz.size(0))
+            z, recons = poutine.replay(air.prior, trace=trace)(examples_to_viz.size(0))
```

Comment:
Can we use positional args when the function is specified? If so the you could shrink this PR by avoiding `trace=` in many places.

Reply:
I guess, but I kind of prefer the explicit keyword arguments as a convention